src/scripts/ci_build.py

```diff
@@ -152,7 +154,12 @@ def determine_flags(target, target_os, target_cpu, target_cc, cc_bin,
         flags += ['--ldflags=-static']
 
     if target == 'coverage':
-        flags += ['--with-coverage-info', '--with-debug-info', '--test-mode', '--terminate-on-asserts']
+        flags += ['--with-coverage-info']
+
+    if target in ['coverage', 'sanitizer']:
+        if target_os != 'windows':
+            flags += ['--with-debug-info']
```

Comment:
> Shall we try it?


I would say yes

Reply:
[Seems to work just fine](https://github.com/randombit/botan/actions/runs/3548877661/jobs/5960581074) without this. Some macOS-based configurations failed as there seems to be some outage at homebrew.

Reply:
You commented out the code, but I think it should be removed

Reply:
Also you now removed `--with-debug-info` now completely. But for nice outputs we probably want debug info in our sanitizer builds.
So I think it should be added for all, not removed for all.

Reply:
[Another bread-crumb](https://stackoverflow.com/a/21855956): Someone had this error while building inside a Dropbox share, because the dropbox daemon locked files at will and interfered with the build. Maybe some anti-virus software is meddling with the file.